src/design/plone/contenttypes/restapi/serializers/unita_organizzativa.py

```diff
@@ -36,10 +41,5 @@ def __call__(self, version=None, include_items=True):
             for x in brains
```

Comment:
c'è un utility di restapi che converte il brain in json con i dati minimi, se vogliamo tenere le cose più standard possibile

Reply:
@cekk visto:

`        servizi = [
            getMultiAdapter((x, self.request), ISerializeToJsonSummary)()
            for x in brains
        ]`

modificato e test ancora verdi